rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -1380,3 +1393,23 @@ game_info game_list_frame::GetGameInfoFromItem(QTableWidgetItem* item)
 
 	return var.value<game_info>();
 }
+
+QColor game_list_frame::getGridCompatibilityColor(const QString string)
```

Comment:
This function is completely unnecessary, since QColor(string) already returns an invalid color if the string is garbage